src/commands/help.js

```diff
@@ -48,7 +50,7 @@ const Module = new Augur.Module().addCommand({
               true
             );
             if (i == 20) {
-              msg.author.send({ embed: embed });
+              msg.author.send({ embed: embed }).catch(console.error);
```

Comment:
Didn't use `u.alertError` as the error might be caused by the discord API, therefore I'd rather have it in the console